src/lib.rs

```diff
@@ -136,6 +140,14 @@ pub fn join_with(cmd_args: Option<&[&str]>) -> Result<(), String> {
     let verbosity = format!("-{}", "v".repeat(2 + args.nodes_verbosity as usize));
     common_args.push(&verbosity);
 
+    let max_capacity_string;
+    if let Some(max_capacity) = args.max_capacity {
+        println!("{:?}", max_capacity);
```

Comment:
Shouldn't we put some label before the capacity number to print out?
I don't think we need to print it though, we don't seem to print any of the other args individually, but if we still want to do it perhaps consider wrapping it in a `if args.verbosity > 0 {` ?

Reply:
My bad, that was just for me to debug,I will remove and push.